src/fshtypes/ValueSetTerm.ts

```diff
@@ -0,0 +1,7 @@
+import { FshCode } from './FshCode';
+
+export class ValueSetTerm extends FshCode {
+  constructor(code: string, system: string, display?: string, public definition?: string) {
```

Comment:
If we pop the `system` up a level, we don't need it here.  Also... I don't think there is anywhere that `definition` comes into play?  It might if we allow defining _new_ codes in a `ValueSet`, but I think we're actually leaning toward not allowing that (you can only use _existing_ codes from an _existing_ value set).